docs/reference/settings/ssl-settings.asciidoc

```diff
@@ -74,6 +74,10 @@ include::{es-repo-dir}/settings/common-defs.asciidoc[tag=ssl-certificate]
 (<<static-cluster-setting,Static>>)
 include::{es-repo-dir}/settings/common-defs.asciidoc[tag=ssl-certificate-authorities]
 
++{ssl-prefix}.ssl.verification_mode+::
+(<<static-cluster-setting,Static>>)
+include::{es-repo-dir}/settings/common-defs.asciidoc[tag=ssl-verification-mode]
+
```

Comment:
Do we need a new block since there is an existing one at line 29 of the same file? The existing block is controlled by `ifdef::verifies[]` and existing HTTP SSL setting section has this variable set to false `:verifies!`. Adding a new block here makes it show up twice for Transport SSL settings.

Reply:
Thanks for pointing out this discrepancy! I changed the `ifdef` statement to `ifdef:server` so that the same definition can be used in both the HTTP and Transport contexts.